deploy/helm/templates/leader_election.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if .Values.rbac.create}}
+---
+# permissions to do leader election.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "trivy-operator.fullname" . }}-leader-election
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "trivy-operator.labels" . | nindent 4 }}
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
```

Comment:
when `leases` was in `ClusterRole` it use to have only verbs (create, get, update) now in `Role` has additional (list, watch, patch, delete).
advice on this change

Reply:
Allowing same verbs on the namespace level now.